app/helpers/admin/republishing_helper.rb

```diff
@@ -68,11 +68,23 @@ def bulk_content_type_metadata
   end
 
   def republishable_content_types
-    editionable_content_types = Edition.descendants.select { |descendant|
-      descendant.descendants.count.zero?
-    }.map(&:to_s)
+    editionable_content_types = %w[
```

Comment:
**thought:** maintainability

This was always a bit meh. The benefit here - assumptions aside - was that if a new type gets added, the developer who adds it doesn't need to remember to add it to the list(s) of republishable content types, but maybe we can just deal with missing types via support requests if and when they come up?

I encountered a different issue with `.descendants` in some of the GraphQL work, but we're considering still including it in test land to ensure we keep a hard-coded list up-to-date: I suppose we could do something similar here to ensure any new leaf nodes get added to this list, but then we'd still not be checking for new non-leaf republishable descendant nodes, so maybe it's too half-baked to be worth it

Reply:
Done in  331793d